### PR TITLE
enable open_curly_linter in .lintr

### LINKED ISSUE #596: Make R/ conform to `open_curly_linter`
Part of #584 

- [ ] enable `open_curly_linter` in `.lintr`
- [ ] run `lintr::lint_package()` to find all lints
- [ ] fix all lints

### PR DESCRIPTION
**Pull request by MichaelChirico**
Closes #596

There are already no lints for this linter, we just need to enable it in the options:

```
lintr::lint_package(linters = lintr::open_curly_linter)
# returns nothing
```

### CODE REVIEW

**Comment by AshesITR** [comment]
This is already part of the default linters, no need to manually add it.

File: .lintr [hunk 1]
```diff
@@ -3,7 +3,7 @@ linters: with_defaults( # The following TODOs are part of an effort to have {lin
    infix_spaces_linter = NULL, # TODO enable (#594)
    commented_code_linter = NULL, # TODO enable (#595)
    cyclocomp_linter = cyclocomp_linter(29), # TODO reduce to 15
-   open_curly_linter = NULL, # TODO enable (#596)
+   open_curly_linter,
```
Comment (AshesITR) [inline .lintr:6]: you can just delete the line. It will be enabled through `with_defaults()`

**Review by AshesITR — CHANGES_REQUESTED** [review]

**Comment by MichaelChirico** [comment]
Hmm, the linter bot seems to have found lints that the command in the issue did not:

```
lintr::lint_package(linters = lintr::open_curly_linter)
```

This also didn't detect those lints that the bot found:

```
for (f in list.files()) print(lintr::lint(f, lintr::open_curly_linter))
```

The lints look correct though, any idea?

**Comment by MichaelChirico** [comment]
Well, I can reproduce locally with this at least:

```
Rscript -e "lintr::lint_package('.')"
```

with the new settings. I'll edit the issue descriptions...

**Review by AshesITR — APPROVED** [review]

**Pull request merged**